src/games/avalon/components/rules.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react'
+import { Typography } from '@material-ui/core'
+import { ActionRow } from '../../../components/actionRow'
+import { Button } from '../../../components/button'
+import { makeStyles } from '@material-ui/styles'
+
+interface Props {
+  done: () => void
```

Comment:
Let's also remove done here

Reply:
Done.

Had to remove some extra code in the component to make this work